src/pages/MixCards.tsx

```diff
@@ -162,12 +166,23 @@ function MixCardsContent() {
                     onChange={handleRadioChangeGames}
                   >
                     {cards.map((card) => (
-                      <FormControlLabel
-                        key={card.id}
-                        value={card.id}
-                        control={<Radio />}
-                        label={card.name}
-                      />
+                      <Accordion>
+                        <AccordionSummary
+                          expandIcon={<ArrowDropDownIcon />}
+                          aria-controls="panel1-content"
+                          id="panel1-header"
+                        >
+                          <Typography>{card.name}</Typography>
+                        </AccordionSummary>
+                        <AccordionDetails>
+                          <FormControlLabel
+                            key={card.id}
+                            value={card.id}
+                            control={<Radio />}
+                            label={card.name}
+                          />
```

Comment:
as soon as here accordion, please add some kind of card to display all values not only the name